lib/authentication/view/login_view.dart

```diff
@@ -141,114 +141,122 @@ class _LoginViewState extends State<LoginView> {
         }
       },
       child: Scaffold(
-        body: Stack(
-          fit: StackFit.expand,
-          children: <Widget>[
-            Align(
-              alignment: FractionalOffset.topRight,
-              child: Padding(
-                padding: const EdgeInsets.all(10),
-                child: Container(
-                    height: MediaQuery.of(context).size.height / 3,
-                    child: Image.asset(
-                        'assets/illustrations/undraw_digital_nomad.png')),
-              ),
-            ),
-            Align(
-              alignment: FractionalOffset.bottomCenter,
-              child: ConstrainedBox(
-                constraints: BoxConstraints(
-                  maxHeight: MediaQuery.of(context).size.height / 4,
-                  maxWidth: MediaQuery.of(context).size.width,
+        body: Center(
```

Comment:
What do changes in this file have to do with this PR? Please make a separate PR if you wanna improve the login page layout.